### DISCUSSION TITLE
Special Alternate Solution

### DISCUSSION THREAD

**Discussion by JohnE1617** [snwau/picoCTF-2023-Writeup#2] [General]
Excellent job with the syntax to get the flag. We were able to escape the program by inputting `;` into the terminal and then executing commands as regular bash.
example
$ s;s;s;s;cd blargh;cat flag.txt;

Great write ups so far

**Comment by snwau** [comment]
Nice one! seems like a more elegant solution than the one I found.
Thank you for sharing, it's great to hear of alternative ways to solve the problems.
